test/test_image.py

```diff
@@ -16,6 +16,12 @@
     decode_png, decode_jpeg, encode_jpeg, write_jpeg, decode_image, read_file,
     encode_png, write_png, write_file, ImageReadMode, read_image)
 
+try:
+    import cv2
```

Comment:
We should remove the dependency on OpenCV on our tests, specially as we can use PIL here.

Reply:
Okay, removed.